service/controller/app/resource/configmap/resource.go

```diff
@@ -86,7 +87,24 @@ func equals(a, b *corev1.ConfigMap) bool {
 	if !reflect.DeepEqual(a.Annotations, b.Annotations) {
 		return false
 	}
-	if !reflect.DeepEqual(a.Data, b.Data) {
+
+	var source, dest map[string]interface{}
+	{
+		source = make(map[string]interface{})
+		dest = make(map[string]interface{})
+
+		err := yaml.Unmarshal([]byte(a.Data["values"]), &source)
+		if err != nil {
+			return false
+		}
+
+		err = yaml.Unmarshal([]byte(b.Data["values"]), &dest)
+		if err != nil {
+			return false
+		}
+	}
+
+	if !reflect.DeepEqual(source, dest) {
```

Comment:
Comparing two values as a struct, not YAML strings would not trigger the constant updates.